src/client/src/rt-theme/themes.ts

```diff
@@ -2,7 +2,7 @@ import { mapValues } from 'lodash'
 
 import colors, { AccentPaletteMap, Color, ColorPaletteMaps, CorePalette, CorePaletteMap } from './colors'
 
-export interface Theme {
+export interface BaseTheme {
```

Comment:
This can just be `ReturnType<typeof createTheme>`

Reply:
For reasons left for last Friday night, it won't work quiet the way you want if it's done that way.